### DISCUSSION TITLE
Can I registrate betweene 2d and 3d.

### DISCUSSION THREAD

**Discussion by timeanddoctor** [Project-MONAI/tutorials#483] [Q&A]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

Can I registrate betweene 2d and 3d.

**Comment by wyli** [comment]
There is no worked example on this, but the resampling, warping component supports this, you can use spatial shape (H, W, 1) in 3d.
